### PR TITLE
Added DuckStation Emulator

### PR DESCRIPTION
**Pull request by EduardTheFirst**
Hey i think i finally managed to do it right after i rewrote the files again !

### CODE REVIEW

**Comment by monyarm** [comment]
Looks to me like you copied the yuzu runner, i'd say it'd be a good idea to remove the prod.keys and title.keys stuff, cause duckstation doesn't need it.

File: lutris/runners/runner.py [hunk 1]
```diff
@@ -371,16 +371,30 @@ def download_and_extract(self, url, dest=None, **opts):
         callback = opts.get("callback")
         tarball_filename = os.path.basename(url)
         runner_archive = os.path.join(settings.CACHE_DIR, tarball_filename)
+        AppImage = "AppImage"
```
Comment (strycore) [inline lutris/runners/runner.py:374]: what is going on here?

File: lutris/runners/runner.py [hunk 2]
```diff
@@ -371,16 +371,30 @@ def download_and_extract(self, url, dest=None, **opts):
...
+            dest = "settings.RUNNER_DIR + tarball_filename"
+            downloader(
+                url, runner_archive, self.run, {
+                    "archive": runner_archive,
+                    "dest": dest,
+                    "merge_single": merge_single,
+                    "callback": callback,
+                }
+            )
+        if not tarball_filename.endswith(AppImage):
```
Comment (strycore) [inline lutris/runners/runner.py:389]: so there's this thing called "else"

File: lutris/runners/runner.py [hunk 3]
```diff
@@ -371,16 +371,30 @@ def download_and_extract(self, url, dest=None, **opts):
...
-                "archive": runner_archive,
-                "dest": dest,
-                "merge_single": merge_single,
-                "callback": callback,
-            }
-        )
+            # Verifying is the link is AppImage and if not it wont Unarchive it
+        if tarball_filename.endswith(AppImage):
+            runner_archive = os.path.join(settings.RUNNER_DIR, tarball_filename)
+            dest = "settings.RUNNER_DIR + tarball_filename"
```
Comment (strycore) [inline lutris/runners/runner.py:380]: that is not going to work.

**Comment by strycore** [comment]
Closing this because there are too many things that go wrong in this PR. Please reopen when ready to merge. Also consider making a JSON runner. There doesn't seem to be any option needed.

**Pull request closed without merging**